.github/workflows/ci.yml

```diff
@@ -0,0 +1,320 @@
+name: Cross Platform Build And Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    if: github.event.pull_request.draft == false
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        vtk_version: [37fe47d99b1b7a802b737fa7d3d87b2d1187927d, v9.0.1]
+
+    runs-on: ${{matrix.os}}
+    container: ${{ matrix.os == 'ubuntu-latest' && 'ghcr.io/f3d-app/f3d-ci' || null }}
+
+    env:
+      DISPLAY: :0
+
+    steps:
+    - name: Dependencies Dir
+      working-directory: ${{github.workspace}}
+      run: mkdir dependencies
+
+    - name: Cache OCCT
+      id: cache-occt
+      uses: actions/cache@v2
+      with:
+        path: dependencies/occt_install
+        key: occt-V7_5_1-${{matrix.os}}
+
+    # Checkout action does not work with non github repositories:
```

Comment:
Do you think we should use the mirror here? https://github.com/Open-Cascade-SAS/OCCT

Reply:
It doesn't contains all the tags for some reasons.

Reply:
it seems to contains 7_6_0, so that could work in that case.